x-pack/test/ingest_manager_api_integration/config.ts

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import path from 'path';
+
+import { FtrConfigProviderContext } from '@kbn/test/types/ftr';
+import { defineDockerServersConfig } from '@kbn/test';
+
+export default async function ({ readConfigFile }: FtrConfigProviderContext) {
+  const xPackAPITestsConfig = await readConfigFile(require.resolve('../api_integration/config.ts'));
+
+  const registryPort: string | undefined = process.env.INGEST_MANAGEMENT_PACKAGE_REGISTRY_PORT;
+
+  // mount the config file for the package registry as well as
+  // the directory containing additional packages into the container
+  const dockerArgs: string[] = [
+    '-v',
+    `${path.join(
+      path.dirname(__filename),
+      './apis/fixtures/package_registry_config.yml'
+    )}:/registry/config.yml`,
+    '-v',
+    `${path.join(
+      path.dirname(__filename),
+      './apis/fixtures/test_packages'
+    )}:/registry/packages/test-packages`,
+  ];
+
+  return {
+    testFiles: [require.resolve('./apis')],
+    servers: xPackAPITestsConfig.get('servers'),
+    dockerServers: defineDockerServersConfig({
+      registry: {
+        enabled: !!registryPort,
+        image: 'docker.elastic.co/package-registry/package-registry:PR-539',
```

Comment:
This might just be my ignorance on the subject of docker images 😬  - what is the `PR-539` at the end? is that the ID of the package (which maybe is derived from the PR that created it?)

Reply:
It is the ID of the PR in `elastic/package-registry` that the image was built from. We need a stable docker image, but it is not decided how we will keep track of the images used for these tests that is both trackable and at least somewhat understandable for the developer. I'm adding some documentation that addresses this, but: good catch.

Reply:
Oh, and we can use anything that is available on https://container-library.elastic.co/r/package-registry/package-registry , and the @elastic/observablt-robots team can help us put the containers there that we need. We "just" need to come up with a plan when to build a container to use for CI, from what branch and commit, and how to name it. (Cc: @ruflin )

Reply:
It was too noisy to publish the PR Docker images on the package-registry namespace, so we have changed the PR images to the observability-ci namespace(https://container-library.elastic.co/r/observability-ci/package-registry) we use the PR-XXXX and the SHA1 as tags, every green build deploy a Docker image. 
If you need anything you just have to ping us someone will manage to help you.

Reply:
thanks for the explanation 😄